models/keyword.go

```diff
@@ -30,15 +30,21 @@ func UpdateKeyword(k *Keyword) (*Keyword, error) {
 	return k, nil
 }
 
-func GetKeywords(u *User) orm.QuerySeter {
+func GetKeywords(u *User, keyword string) orm.QuerySeter {
 	orm := orm.NewOrm()
-	return orm.QueryTable(Keyword{}).Filter("user_id", u.Id)
+	result :=  orm.QueryTable(Keyword{}).Filter("user_id", u.Id)
+
+	if len(keyword) > 0 {
+		result = result.Filter("name__icontains", keyword)
+	}
+
+	return result
 }
 
-func GetPaginatedKeywords(u *User, offset int, sizePerPage int) ([]*Keyword, error) {
+func GetPaginatedKeywords(keywords orm.QuerySeter, offset int, sizePerPage int) ([]*Keyword, error) {
```

Comment:
@junan Should we improve this function like this from BeeGo?
https://github.com/beego/bee/blob/develop/generate/g_model.go#L178

It looks flexible to use 😛

Reply:
Actually filtered keywords already, here I am just paginating them. Current one ok to me. What do you think?

Reply:
Or we should update the function name from `GetKeywords` to `GetKeywordQuerySeter`?
Because the return function is about `QuerySeter`.
I and @gutakk have the same confusing of its name that it should return with list of keyword not the queryseter 🤔

Reply:
Hmm, `GetKeywords` function actually returns keywords just it return as `QuerySeter` type, its kind of same to Rails where it returns as `ActiveRecordRelations` instead of array. But we dont named the function/method based on return type like `def get_active_record_relations`. What is your thought?

Reply:
Based our discussion, I renamed to `GetQuerySeterKeywords`